### PR TITLE
Fix and add connect() around displayName

### PR DESCRIPTION
**Pull request by odinho**
Sadly the fix in #2 didn't actually work, since it was using the name of React's `Component` instead of the actual component `Comp` being taken in.

The second commit also decorates the component displayName with `Connected()`, so it is simpler to see whether you are using a connected or unconnected component in your tests, dom snapshots or elsewhere. This commit is optional in this PR, I can remove it if you are against it. But I think it makes sense.

I also wrote some tests to show how it works and actually make sure it *does* work this time around.

### CODE REVIEW

**Comment by odinho** [comment]
So some self-review on this.

I was a bit iffy about adding the extra `Connected(MyComponent)` around the `displayName`. Is it really required? As you can tell, I did land on "yeah, it's cost is quite small, and usefulness if you do use this name for anything is quite high".
About the name itself, I've been thinking that maybe `Connected` might not be the right word to decorate a connected component with. The exported name we use to connect to redux-bundler is `connect`.

So maybe `connect(MyComponent)` would make more sense?

@HenrikJoreteg Sorry I didn't answer your help for maintenance question. I'm trying to make my team see the light with this, and just had to fix some issues. I could indeed help out with maintaining, but it's mostly dependent on if I'm able to make redux-bundler stick in our team or not. Privately I do not use React (I'd prefer vanilla JS and web components potentially with HyperHTML or somesuch for virtualdom).

@huygn @aulneau You guys are maintainers of this too I saw in your PRs, maybe you'd like to review this?

(On a side note, is there a list where you can see who are the maintainers? I didn't find it)

**Comment by odinho** [comment]
@HenrikJoreteg I can help maintain this project. I see that there's some man-power issues here : )

It'd be good to get this fix in. We use the displayName quite a lot in printouts of the different expected trees (using jest snapshots). And since more and more React components are now redux-bundler connected, they all revert to having `<e />` as their name, which is a bit meh. :)

File: test/react-bindings.js [hunk 1]
```diff
@@ -19,7 +19,9 @@ const getStore = composeBundlesRaw({
   selectIsPositive: createSelector('selectNumber', number => number > 0)
 })
 
-const Page = ({ number }) => createElement('div', null, number)
+const Page = ({ number }) => {
+    return createElement('div', null, number)
+}
```
Comment (odinho) [inline test/react-bindings.js]: ??? Don't know how this got here. Fix before merge.

**Comment by odinho** [comment]
@HenrikJoreteg @huygn @aulneau  Ping.

**Comment by HenrikJoreteg** [comment]
Hi @odinho! Sorry this has been sitting for so long! You're right there's some manpower issues :) 

Personally, I use Preact for everything so I don't do much with this lib. If you're willing to help maintain this I'm happy to give you access (in fact i just did). I gave you npm perms too. 

This looks like a pretty harmless little fix. I say ship it :)

**Pull request merged**